plugins/logitech-rallysystem/fu-logitech-rallysystem-plugin.c

```diff
@@ -12,6 +12,8 @@
 
 struct _FuLogitechRallysystemPlugin {
 	FuPlugin parent_instance;
+	gchar *system_version;
```

Comment:
same concern as @superm1 -- what is you have two devices inserted? Maybe we can have a hash table of serial_number to system_version so that you can have multiple ones inserted without chaos.